File: FrontendWebApp/web-shop-app/src/models/Product.js
export class Product {
    constructor(
      ProductId=0,
      Name="",
      Description="",
      Price=0,
      Quantity=1,
      Image="",
      SellerId=0,
      
    ) {
      this.productId=ProductId;
      this.name=Name;
      this.description=Description;
      this.price=Price;
      this.quantity=Quantity;
      this.image=Image;
      this.sellerId = SellerId;
    }
  }
  export default Product;

File: BackendWebApp/WebShopAPI/WebShopAPI.Models/Product.cs
namespace WebShopAPI.Models
{
    public class Product
    {
        public int ProductId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        public string Image { get; set; }
        public int SellerId { get; set; }
        //public User Seller { get; set; }
        
    }
}
